src/utilities/exitHandling.ts

```diff
@@ -4,6 +4,6 @@ export function configureExitHandling(logger: Logger): void {
   process.on('uncaughtException', (err) => {
     logger.fatal({ err }, 'uncaughtException');
 
-    process.exit(1);
+    process.exitCode = 1;
```

Comment:
This approach is recommended over the previous one: https://nodejs.org/api/process.html#event-uncaughtexception